templates/footer.html

```diff
@@ -0,0 +1,63 @@
+<footer class="p-strip--suru-top has-accent is-dark {% if request.path == '/' %}is-homepage{% endif %}"
+  style="background-size: auto 100%, cover; background-image: url('https://assets.ubuntu.com/v1/8951dc67-suru-left.svg'), linear-gradient(266deg, #044f66, #022935)">
+  <div class="row">
+    <div class="col-8">
+      <h3>The Operator Framework</h3>
+      <p>The Operator Framework provides a simple, lightweight, and powerful way of writing Juju charms, the best way to encapsulate operational experience in code.</p>
+      <ul class="p-inline-list u-sv3">
+        <li class="p-inline-list__item"><a href="https://github.com/canonical/operator/" class="p-button--neutral">Get
+            involved</a></li>
+        <li class="p-inline-list__item"><a href="https://charmhub.io/#get-in-touch" class="p-button--outline">Contact us</a></li>
```

Comment:
This is a confusing button on Charmhub too. What are you contacted. Framework team? Charmhub? Maybe leave this button off for now?

Reply:
Ok, done.